power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/linear_pf_solver.hpp

```diff
@@ -113,13 +115,16 @@ template <bool sym> class LinearPFSolver {
     }
 
     void calculate_result(YBus<sym> const& y_bus, PowerFlowInput<sym> const& input, MathOutput<sym>& output) {
-        common_solver_functions::calculate_result(y_bus, input, *sources_per_bus_, *load_gens_per_bus_, output,
-                                                  [](Idx /*i*/) { return LoadGenType::const_y; });
+        detail::calculate_result(y_bus, input, *sources_per_bus_, *load_gens_per_bus_, output,
+                                 [](Idx /*i*/) { return LoadGenType::const_y; });
     }
 };
 
 template class LinearPFSolver<true>;
 template class LinearPFSolver<false>;
+} // namespace math_solver::linear_pf
+
+using math_solver::linear_pf::LinearPFSolver;
 
 } // namespace power_grid_model
```

Comment:
this is in the wrong namespace. should be:

```suggestion
using linear_pf::LinearPFSolver;

} // namespace power_grid_model::math_solver
```